charts/eks-anywhere-packages/templates/registrymirrorsecret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
+  {{- with .Values.registryMirrorSecret }}
+  ENDPOINT: "{{ .endpoint | b64enc }}"
+  USERNAME: "{{ .username | b64enc }}"
+  PASSWORD: "{{ .password | b64enc }}"
+  CACERTCONTENT: "{{ .cacertcontent | b64enc }}"
```

Comment:
These should be base64 encoded when they are put in the values.yaml or we are going to have problems.

Reply:
On the cert, I think it makes sense to base64 encode it for the newlines rather than do a special space replace.